digital_bazaar.js

```diff
@@ -6,17 +6,22 @@ import {
 	uploadImageToGoogleDrive,
 } from './dist/index.js';
 
-const accessToken = 'FIRST_ACCESS_TOKEN';
-const credentialEngine = new CredentialEngine(accessToken);
+const accessTokenA =
+	'ya29.a0AeDClZC35NOgFhg8kz2OzOKpMEAQeNkYydmmYUJLnYwPRAd2PUH0_yWqs2ql2Q8vq67IK4G7ATqOhOvSJlnEgV7t8ClNS9VxGMXgvA9473EutBZC-XXmzDIzC3qHTOI4aB_chVyNWWTaJSjBKc7xamuDBcwgjUKp3WbKLp_faCgYKAawSARESFQHGX2MiJxVd7XC-tzE5L26cvN97IA0175';
+const accessTokenB =
```

Comment:
this should not be in code we are checking in to a shared public repo @0marSalah